tests/backup/backup_test.go

```diff
@@ -340,6 +617,71 @@ func DeleteS3Bucket(bucketName string) {
 		fmt.Sprintf("Failed to delete bucket [%v]. Error: [%v]", bucketName, err))
 }
 
+// CreateCloudCredential creates cloud credetials
+func CreateCloudCredential(provider, name string, orgID string) {
+	Step(fmt.Sprintf("Create cloud credential [%s] in org [%s]", name, orgID), func() {
+		logrus.Printf("Create credential name %s for org %s provider %s", name, orgID, provider)
+		backupDriver := Inst().Backup
+		switch provider {
+		case providerAws:
+			log.Printf("Create creds for aws")
+			id := os.Getenv("AWS_ACCESS_KEY_ID")
+			Expect(id).NotTo(Equal(""),
```

Comment:
Can we use the function `getAWSDetailsFromEnv()` as being used in master?
https://github.com/portworx/torpedo/blob/master/tests/backup/backup_test.go#L381